nautilus_trader/adapters/sandbox/execution.py

```diff
@@ -109,7 +109,7 @@ def __init__(
         )
         self.exchange = SimulatedExchange(
             venue=sandbox_venue,
-            oms_type=oms_type,
+            oms_type=OmsType.HEDGING if oms_type == OmsType.NETTING else OmsType.NETTING,
```

Comment:
So the oms_type is changed to hedging if netting is used? Feels a bit off but maybe I understand this wrong.

Reply:
I believe either one of `SimulatedExchange` or `ExecutionClient` should be generating the `VenueId`. When we use `NETTING` for both None of them is generating IDs and alternatively both generate the IDs which doesn't work.